packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/AddPrayerFormConnected.js

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';
+import { Query } from 'react-apollo';
+import { get } from 'lodash';
+import getProfilePic from '../../getProfilePic';
+import AddPrayerForm from './AddPrayerForm';
+
+const AddPrayerFormConnected = memo(({ ...props }) => (
+  <Query query={getProfilePic} fetchPolicy={'cache-and-network'}>
```

Comment:
I don't think this query is actually needed. We have the current user with their profile pic saved to cache. We shoul dbe able to pull that from cache.